changelog/fragments/add-deafult-olm-version.yaml

```diff
@@ -0,0 +1,7 @@
+entries:
+  - description: >
+      Include a default version of OLM manifests with SDK. Add Makefile target to enable users to 
```

Comment:
The `Makefile target` added is SDK and the user's projects scaffolded was not changed at all. So, this description is not accurate.